core/engine/src/context/mod.rs

```diff
@@ -106,7 +108,7 @@ pub struct Context {
     can_block: bool,
 
     #[cfg(feature = "temporal")]
-    tz_provider: FsTzdbProvider,
+    tz_provider: CompiledTzdbProvider,
```

Comment:
Does this make the engine quite heavy binary wise? Id expect to see the runtime pass in data rather than the engine use compiledData directly. That being said i'd imagine tests would break if we have no dta when testing the engine? (maybe i don't understand this)

Reply:
Not particularly so. We will always have to include in some sort of data that will need to be packaged or compiled in because of windows. My plan is to expose this to be a configurable option in the future.